contributing/node-versions.md

```diff
@@ -1,6 +1,8 @@
 # Node Versions
 
-The site currently runs on Node.js v14, the [Active LTS version](https://nodejs.org/en/about/releases/) from 2020-10-27 to 2021-10-26.
+In [development](contributing/development.md) enviroments this site will run on Node.js versions `12 - 14`.
+
+In [staging and production](contributing/deployments.md) environments this site runs on Node.js 14, the [Active LTS version](https://nodejs.org/en/about/releases/) from 2020-10-27 to 2021-10-26).
```

Comment:
it's a bit odd to say that Node 14 is the Active LTS when it's not. Perhaps just remove the qualification and say "this site runs on the latest in the Node.js 14 family"  (or Node.js 14.* as undoubtedly there will be a gap between a new 14.* coming out and us deploying).

Reply:
It's not the Active LTS now, but it will be in 3 weeks. Hence the `from 2020-10-27 to 2021-10-26.`